packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -99,6 +99,7 @@ _The key elements of the component's public API surface:_
     -   `colorScaleMode` - represent an Enum value that determent if the colorScale is represent a continues gradient values (linear), or is set categorically (ordinal).
```

Comment:
Reviewed with @munteannatan and discussed not exposing this property as a public API to enforce consistency across apps rendering wafermaps. Could not identify a use-case for having the die margin > 0 yet but if it changes it should be consistent across usages.